CHANGELOG.md

```diff
@@ -4,9 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## 0.1.3 - 20178-02-08
+## 0.2.0 - 2018-03-14
+Interaction release
+
+### Added
+* New `clearUploads()` method to remove current ulpoads.
+* New `reset()` method reset widget mannualy.
```

Comment:
Well, no need to add the "New" thing here, since the heading says the methods were added (which implies they weren't there before).

* `clearUploads()`, a method that removes all current uploads from the widget: "clears" it.
* `reset()`, a method allowing for the manual widget reset.

Regarding the widget reset, this could use a link to the docs.

Reply:
?? Note, when you remove current uploads from a widget, the files are not removed from your UC project defined by its public key.